### ISSUE TITLE
User input not always black

### ISSUE THREAD

**Issue by JeffreyBenjaminBrown** [joshsh/brain-mode#18]
If unpushed text was always black, it would be visibly apparent whether a buffer has unpushed changes.

**Comment by joshsh** [comment]
Emacs likes to extend the properties at point to new text typed after point.  If we can disable this feature, it would indeed make changes more apparent.

**Comment by JeffreyBenjaminBrown** [comment]
I googled for a while, found nothing, [posted to stackoverflow](https://stackoverflow.com/questions/42935147/emacs-lisp-disable-extension-of-surrounding-text-properties-to-new-text).

**Comment by joshsh** [comment]
Well, I have determined that the following does *not* help:

```
(setq text-property-default-nonsticky '((foreground . t)))
```

**Comment by JeffreyBenjaminBrown** [comment]
lol

**Comment by JeffreyBenjaminBrown** [comment]
It's doable! Someone on Stackoverflow pointed to [this](https://www.gnu.org/software/emacs/manual/html_node/elisp/Sticky-Properties.html). Each property's front- and rear-stickiness can be set. The easiest solution seems to be to make them all not sticky.

**Comment by joshsh** [comment]
Yep.  I saw that and tried what I mentioned.  It didn't "stick".  Possibly something to do with the scope of the variable.
